src/App.js

```diff
@@ -13,8 +13,12 @@ function App(props) {
     return <ProductDetail key={pro.id} product={pro} />
   })
   return(
```

Comment:
Looks like you might need to start over the App.js
The intention was to only take out the html to make the other components. It looks like you took out all of the html
Other then that, it looks like you have everything else done properly. you just need to import the components and use them in App.js